### ISSUE TITLE
Include rows that indicate whether a variable was included (i.e., "Yes" or "No")

### ISSUE THREAD

**Issue by leifeld** [leifeld/texreg#10]
Comment by user [felix_h](https://r-forge.r-project.org/users/felix_h/) (Felix Haas) on R-Forge:

My second relates to custom additional rows. Sometimes it is desirable to indicate whether some set of variables was included in a model but not shown (e.g. country fixed-effects), see e.g. the "Time Dummies" row in this table:
![msvnq](https://cloud.githubusercontent.com/assets/5451293/20846006/8ec10e6e-b8bf-11e6-908f-a9f7cf37f4e8.png)

Is it possible to add a similar functionality to ``texreg``?

Again thanks for such a great package and keep up the good work!

Best
Felix

**Comment by alanpaulkwan** [comment]
I have modified screenreg/texreg to do this. May I share? It is based on an old version of texreg but should be obvious how to integrate.

**Comment by leifeld** [comment]
@alanpaulkwan Yes, I would appreciate it!

**Comment by alanpaulkwan** [comment]
https://www.dropbox.com/s/ryy1lbf9ji409s6/texreg.mine.R?dl=0

Part of me thinks its as simple as adding "add.lines" and then the call to
"add.lines" in each function (search add.lines). Part of me also remembers,
vaguely, that add.lines is a character and the gof.matrix is a numeric
matrix. I forget how I dealt with that, if at all. Should be easy to cast
at some point, in any case.

Feel free to call it something else, it's your show :)

**Comment by leifeld** [comment]
Thanks. Here is a persistent copy in case your Dropbox resource is removed: [texreg.mine.txt](https://github.com/leifeld/texreg/files/628690/texreg.mine.txt)

I (or somebody else) may need to do a diff comparison at some point.

Yes, the numeric vs. character thing may be an issue. Perhaps wrap in a `multicolumn`?

**Comment by leifeld** [comment]
See also #54

**Comment by helix123** [comment]
Good idea! Another extension might be: Sometimes one excludes a lot variables and then don't want to "pollute" the output with a line for each. Thus, maybe a facility to put the non-printed coeffcients in the notes (per model) would be great!

**Comment by zauster** [comment]
+1 
stargazer handles it like this: 
1. omit coefficients by regex
2. if there is a label of the regex: if the regex is found and thus coefficients are omitted, include a line in the table that states this omission

**Comment by AndreWal** [comment]
Any news on the issue regarding additional rows containing characters?

**Comment by leifeld** [comment]
Commit 29c7189d8492c0d940f7f25044ba557d247d93fc introduces an argument `custom.gof.rows`, which accepts a named list of row vectors to be inserted at the beginning of the GOF block of the table. For example, `custom.gof.rows = list(A = 12:13, B = c("yes", "no"))` would add two rows with labels `A` and `B`. I have done some research and found that including dummy variable rows for fixed effects and control variables are usually inserted at the beginning of the GOF block, as a matter of good style. The user can still use the `reorder.gof` argument to re-order the rows of the GOF block, including those custom new rows. I hope folks will be happy with this solution. It was fairly hard to implement because it cuts across many different parts of the package, mostly because character strings are usually not permitted as cell contents by design. It will take a while until this release will make it to CRAN because we have a number of other issues to fix. But hang in there.

**Comment by ala-alrababah** [comment]
Does this still work? I tried using custom.gof.rows before and after updating the package and I tried installing it from github as well but nothing seems to work

**Comment by JackLandry** [comment]
Agreed this does not seem updated @ala-alrababah @leifeld Would be an amazing feature!
